### PR TITLE
feat: added dark theme, fixed few components on dark theme

### PR DESCRIPTION
**Pull request by CzarekDryl**
Implemented dark mode, fixed components that were broken on dark mode.

### CODE REVIEW

**Review by KatarzynaStefanowiczPagepro — APPROVED** [review]

File: src/components/frame/Extensions/themes/enum.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export enum Themes {
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/themes/enum.ts:1]: Theme

File: src/components/frame/Extensions/themes/index.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { light, dark } from './consts';
+import { IThemes } from './types';
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/themes/index.ts:2]: ITheme

File: src/components/frame/Extensions/themes/index.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+export const themes: IThemes = {
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/themes/index.ts:4]: theme

File: src/components/frame/Extensions/themes/enum.ts (hunk 1 above)
Comment (CzarekDryl) [reply src/components/frame/Extensions/themes/enum.ts]: Changed

File: src/components/frame/Extensions/themes/index.ts (hunk 3 above)
Comment (CzarekDryl) [reply src/components/frame/Extensions/themes/index.ts]: Changed to `themeList`, I think it's more precise

File: src/components/frame/Extensions/themes/index.ts (hunk 2 above)
Comment (CzarekDryl) [reply src/components/frame/Extensions/themes/index.ts:2]: Same as comment under

**Review by krawczykkatarzyna — APPROVED** [review]

**Pull request merged**